pkgs/games/anki/bin.nix

```
@@ -57,7 +57,7 @@ let
     targetPkgs = pkgs: (with pkgs; [ xorg.libxkbfile krb5 ]);
 
     runScript = writeShellScript "anki-wrapper.sh" ''
-      exec ${unpacked}/bin/anki
+      exec ${unpacked}/bin/anki ${ lib.strings.escapeShellArg commandLineArgs }
```

You call the argument `commandLineArgs` (plural) though use the singular form of `escapeShellArg`, this will result in `"--foo --bar"` being interpreted as a single argument, as if it was wrapped in singlequotes on bash.

You probably want

```suggestion
      exec ${unpacked}/bin/anki ${ lib.strings.escapeShellArgs commandLineArgs }
```